balloc/balloc.c

```diff
@@ -156,6 +156,11 @@ static bool is_spare(uint64_t alloc_flags);
 static bool is_normal(uint64_t alloc_flags);
 static bool is_any(uint64_t alloc_flag);
 
+static int index_hash(struct m0_balloc_super_block *sb,
+		      m0_bcount_t                   block_offset)
```

Comment:
To be more specific `m0_bcount_t  block_offset` can be replaced with `m0_bindex_t   ext_end`

Reply:
done..